doc/sidebar.html

```diff
@@ -221,6 +221,12 @@
           </ul>
         </details>
       </li>
+      
+      <li>
+        <a href="LINKROOTindex.html#mlpack-on-embedded-systems">
+          Embedded Systems
```

Comment:
Up to you, maybe `Embedded Applications` is a better word to use here?  I definitely agree it should be directly linked to from the sidebar.